ci/steps/notify.py

```diff
@@ -66,38 +114,64 @@ def send_notification(
         else:
             status_symbol = '?'
 
-        result_table += f'<tr><td class="align_left">{task}</td><td>{status_symbol}</td></tr>'
+        html_result_table += f'<tr><td class="align_left">{task}</td><td>{status_symbol}</td></tr>'
+
+    txt_result_table = ""
+    for task, status in status_dict.items():
+        txt_result_table += f'Task: {task}\nStatus: {status}\n'
 
     subject = f'Tekton Pipeline Garden Linux build failure in {pipeline_name}'
     cicd_cfg = glci.util.cicd_cfg(cfg_name=cicd_cfg_name)
     email_cfg = _email_cfg(cicd_cfg=cicd_cfg)
 
-    template_path = os.path.abspath(os.path.join(repo_dir,"ci/templates/email_notification.html"))
-    with open(template_path, 'r') as mail_template_file:
-        mail_template = mail_template_file.read()
+    html_template_path = os.path.abspath(
+        os.path.join(repo_dir, "ci/templates/email_notification.html")
```

Comment:
Agreed. But this would result in a larger refactoring touching many of the files. Should be done in a separate PR. Exceeds by far the scope of this PR.